MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -32,30 +32,40 @@ function build_docker_images() {
 
 function setup_env() {
     export host_ip=${ip_address}
-    export EMBEDDER_PORT=6006
-    export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMBEDDER_PORT/v1/encode"
-    export MM_EMBEDDING_PORT_MICROSERVICE=6000
-    export ASR_ENDPOINT=http://$host_ip:7066
-    export ASR_SERVICE_PORT=3001
-    export ASR_SERVICE_ENDPOINT="http://${host_ip}:${ASR_SERVICE_PORT}/v1/audio/transcriptions"
-    export REDIS_URL="redis://${host_ip}:6379"
-    export REDIS_HOST=${host_ip}
-    export INDEX_NAME="mm-rag-redis"
-    export LLAVA_SERVER_PORT=8399
-    export LVM_ENDPOINT="http://${host_ip}:8399"
-    export EMBEDDING_MODEL_ID="BridgeTower/bridgetower-large-itm-mlm-itc"
-    export LVM_MODEL_ID="llava-hf/llava-v1.6-vicuna-7b-hf"
-    export WHISPER_MODEL="base"
     export MM_EMBEDDING_SERVICE_HOST_IP=${host_ip}
     export MM_RETRIEVER_SERVICE_HOST_IP=${host_ip}
     export LVM_SERVICE_HOST_IP=${host_ip}
     export MEGA_SERVICE_HOST_IP=${host_ip}
-    export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/multimodalqna"
-    export DATAPREP_INGEST_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/ingest_with_text"
-    export DATAPREP_GEN_TRANSCRIPT_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/generate_transcripts"
-    export DATAPREP_GEN_CAPTION_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/generate_captions"
-    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_files"
-    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_files"
+    export REDIS_DB_PORT=6379
+    export REDIS_INSIGHTS_PORT=8001
+    export REDIS_URL="redis://${host_ip}:${REDIS_DB_PORT}"
+    export REDIS_HOST=${host_ip}
+    export INDEX_NAME="mm-rag-redis"
+    export WHISPER_PORT=7066
+    export WHISPER_MODEL="base"
+    export ASR_ENDPOINT=http://$host_ip:$WHISPER_PORT
+    export ASR_PORT=9099
+    export ASR_SERVICE_PORT=3001
+    export ASR_SERVICE_ENDPOINT="http://${host_ip}:${ASR_SERVICE_PORT}/v1/audio/transcriptions"
+    export DATAPREP_MMR_PORT=6007
+    export DATAPREP_INGEST_SERVICE_ENDPOINT="http://${host_ip}:${DATAPREP_MMR_PORT}/v1/ingest_with_text"
+    export DATAPREP_GEN_TRANSCRIPT_SERVICE_ENDPOINT="http://${host_ip}:${DATAPREP_MMR_PORT}/v1/generate_transcripts"
+    export DATAPREP_GEN_CAPTION_SERVICE_ENDPOINT="http://${host_ip}:${DATAPREP_MMR_PORT}/v1/generate_captions"
+    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:${DATAPREP_MMR_PORT}/v1/dataprep/get_files"
+    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:${DATAPREP_MMR_PORT}/v1/dataprep/delete_files"
+    export EMM_BRIDGETOWER_PORT=6006
+    export EMBEDDING_MODEL_ID="BridgeTower/bridgetower-large-itm-mlm-itc"
+    export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMM_BRIDGETOWER_PORT/v1/encode"
+    export MM_EMBEDDING_PORT_MICROSERVICE=6000
+    export REDIS_RETREIEVER_PORT=7000
+    export TGI_GAUDI_PORT="8399:80"
```

Comment:
```suggestion
    export TGI_GAUDI_PORT="${LLAVA_SERVER_PORT}:80"
```
(And move this to be after `LLAVA_SERVER_PORT` is defined)